### PR TITLE
added contributing document

### LINKED ISSUE #75: Create Contribution.md
We need to define basic rules on how to contribute to the repo.

### PR DESCRIPTION
**Pull request by tyv**
we agreed on fact that we need some bureaucracy  in how we are handling our issues.

Please @giladgrush @LeonidLevi @maksymc take a look.

closes #75

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Contributing to Stylable Components
+
+## Branch naming
+We use scheme in branch naming.
+It should contain branch author user name at first and branch name divided by `/`, and optional number of the issue. All words in branch name are separated by underscore: `_`
+`username/own_theme_park_42`
```
Comment (Ta4i) [inline CONTRIBUTING.md]: what if `username/own_theme_park/42` ?

Comment (tyv) [inline CONTRIBUTING.md]: i don't have strong opinion

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+`issue#42`
+**Title**: Build Our Own Theme Park
+**Description**: We need to build own theme park, because at the current park we don't have Blackjack.
+
+#### Pull Request:
+**Title**: Our own Theme park
+**Description**: This PR contains our own theme park implementation, which contains `Blackjack` module and closes #42
+
+## Merging PR's
+We love to squash and merge PR's, but doing that we are using **meaningful** merge commit messages.
```
Comment (wtfil) [inline CONTRIBUTING.md]: does github keep the link to PR on squashed merges?

Comment (tyv) [inline CONTRIBUTING.md]: what do u mean?

File: CONTRIBUTING.md (hunk 1 above)
Comment (JoMarton) [inline CONTRIBUTING.md]: I like `username/issue_num_if_exists/branch_name`... much easier for me to find branches that way

Comment (tyv) [inline CONTRIBUTING.md]: updated

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+Bad is to keep commits history, because most likely it will be like
+
+### Bad example:
```
Comment (JoMarton) [inline CONTRIBUTING.md]: I don't see a need for a bad example. I know I sometimes I am the bad example... but I promise to improve :)

Comment (tyv) [inline CONTRIBUTING.md]: I am thinking this will be removed as soon as everyone will be comfortable with using good.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+# Contributing to Stylable Components
+
+## Branch naming
+We use scheme in branch naming.
+It should contain branch author user name at first and branch name divided by `/`, and optional number of the issue. All words in branch name are separated by underscore: `_`
+`username/own_theme_park/42`
```
Comment (JoMarton) [inline CONTRIBUTING.md]: I myself prefer `username/issue_number_if_relevant/branch_name`... easier for me to find branches that way (if I know who made it and the issue number, I don't need to know how he named the branch)

**Comment by tyv** [comment]
@JoMarton thanks, asked for theirs help

**Pull request merged**